src/main/java/org/ecocompass/core/Main.java

```diff
@@ -6,15 +6,17 @@
 import org.json.JSONArray;
 
 import java.util.List;
+import java.util.Map;
 
 public class Main {
     public static void main(String[] args) throws Exception {
         System.out.println("Ecocompass Core Engine");
-
         Overpass overpass = new Overpass();
-//         String response = overpass.queryLocation("IE", "Dublin", "Rathmines");
-//         overpass.saveQueryOutput(response, "query_data.json");
-        String response = overpass.loadSavedQueryOutput("query_data.json");
+
+        String response = overpass.queryLocation(Map.of("country", "IE", "county", "County Dublin", "city", "Dublin"));
+        overpass.saveQueryOutput(response, "query_data.json");
+//        String response = overpass.loadSavedQueryOutput("query_data.json");
```

Comment:
I am not sure why this is commented out. Okay maybe this is for the saved responses and saved routes. Correct me if I am wrong.